### PR TITLE
Fix up headings in distributed tracing

### PR DESCRIPTION
**Pull request by gewarren**

### CODE REVIEW

File: docs/core/diagnostics/distributed-tracing-collection-walkthroughs.md [hunk 1]
```diff
@@ -192,17 +192,15 @@ For more information about configuring OpenTelemetry, see the [OpenTelemetry get
 
 ## Collect traces using Application Insights
 
-Distributed tracing telemetry is automatically captured after configuring the Application Insights SDK
-([ASP.NET](/azure/azure-monitor/app/asp-net), [ASP.NET Core](/azure/azure-monitor/app/asp-net-core))
+Distributed tracing telemetry is automatically captured after configuring the Application Insights SDK for
+([ASP.NET](/azure/azure-monitor/app/asp-net) or [ASP.NET Core](/azure/azure-monitor/app/asp-net-core)),
 or by enabling [code-less instrumentation](/azure/azure-monitor/app/codeless-overview).
```
Comment (Youssef1313) [inline docs/core/diagnostics/distributed-tracing-collection-walkthroughs.md:197]: Are the parentheses needed here?

**Review by BillWagner — APPROVED** [review]
This LGTM @gewarren 

You can :shipit: when ready.

File: docs/core/diagnostics/distributed-tracing-collection-walkthroughs.md [hunk 2]
```diff
@@ -192,17 +192,15 @@ For more information about configuring OpenTelemetry, see the [OpenTelemetry get
 
 ## Collect traces using Application Insights
 
-Distributed tracing telemetry is automatically captured after configuring the Application Insights SDK
-([ASP.NET](/azure/azure-monitor/app/asp-net), [ASP.NET Core](/azure/azure-monitor/app/asp-net-core))
+Distributed tracing telemetry is automatically captured after configuring the Application Insights SDK for
+([ASP.NET](/azure/azure-monitor/app/asp-net) or [ASP.NET Core](/azure/azure-monitor/app/asp-net-core)),
```
Comment (gewarren) [inline docs/core/diagnostics/distributed-tracing-collection-walkthroughs.md:196]:
```suggestion
[ASP.NET](/azure/azure-monitor/app/asp-net) or [ASP.NET Core](/azure/azure-monitor/app/asp-net-core),
```

File: docs/core/diagnostics/distributed-tracing-collection-walkthroughs.md (hunk 1 above)
Comment (gewarren) [reply docs/core/diagnostics/distributed-tracing-collection-walkthroughs.md]: Good catch, thanks.

**Pull request merged**